src/pages/NotFound.vue

```diff
@@ -3,7 +3,11 @@
     <h1>{{ $t('notFound.pageNotFound') }}</h1>
     <p>{{ $t('notFound.sorry') }}</p>
     <router-link :to="{ name: 'Home' }">
-      <PvButton :label="$t('notFound.label')" icon="pi pi-home" />
+      <PvButton
+        :label="$t('notFound.label')"
```

Comment:
Could I suggest that the notFound.label be replaced with `Return to Home`? The original text is a bit wordy.

Reply:
Oh I can change that, I have to also change the Spanish label

Reply:
Thank you!